File: Bridge/System/Threading/EventWaitHandle.cs
using Bridge;
using System.Security.AccessControl;

namespace System.Threading
{
    public class EventWaitHandle : WaitHandle
    {
        static bool IsManualReset(EventResetMode mode)
        {
            throw new NotImplementedException();
        }

        public EventWaitHandle(bool initialState, EventResetMode mode)
        {
        }

        public EventWaitHandle(bool initialState, EventResetMode mode, string name)
        {
        }

        public EventWaitHandle(bool initialState, EventResetMode mode, string name, out bool createdNew)
        {
            throw new NotImplementedException();
        }

        public EventWaitHandle(bool initialState, EventResetMode mode, string name, out bool createdNew, EventWaitHandleSecurity eventSecurity)
        {
            throw new NotImplementedException();
        }

        public EventWaitHandleSecurity GetAccessControl()
        {
            throw new NotImplementedException();
        }

        public static EventWaitHandle OpenExisting(string name)
        {
            throw new NotImplementedException();
        }

        public static EventWaitHandle OpenExisting(string name, EventWaitHandleRights rights)
        {
            throw new NotImplementedException();
        }

        public bool Reset()
        {
            throw new NotImplementedException();
        }

        public bool Set()
        {
            throw new NotImplementedException();
        }

        public void SetAccessControl(EventWaitHandleSecurity eventSecurity)
        {
            throw new NotImplementedException();
        }
    }
}

File: Bridge/Resources/.generated/system/threading/eventWaitHandle.js
    Bridge.define("System.Threading.EventWaitHandle", {
        inherits: [System.Threading.WaitHandle],
        statics: {
            methods: {
                isManualReset: function (mode) {
                    throw new System.NotImplementedException();
                },
                openExisting: function (name) {
                    throw new System.NotImplementedException();
                },
                openExisting$1: function (name, rights) {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function (initialState, mode) {
                this.$initialize();
                System.Threading.WaitHandle.ctor.call(this);
            },
            $ctor1: function (initialState, mode, name) {
                this.$initialize();
                System.Threading.WaitHandle.ctor.call(this);
            },
            $ctor2: function (initialState, mode, name, createdNew) {
                this.$initialize();
                System.Threading.WaitHandle.ctor.call(this);
                throw new System.NotImplementedException();
            },
            $ctor3: function (initialState, mode, name, createdNew, eventSecurity) {
                this.$initialize();
                System.Threading.WaitHandle.ctor.call(this);
                throw new System.NotImplementedException();
            }
        },
        methods: {
            getAccessControl: function () {
                throw new System.NotImplementedException();
            },
            reset: function () {
                throw new System.NotImplementedException();
            },
            set: function () {
                throw new System.NotImplementedException();
            },
            setAccessControl: function (eventSecurity) {
                throw new System.NotImplementedException();
            }
        }
    });
